src/components/portfolio_elements/PortfolioPreview.vue

```diff
@@ -1,22 +1,64 @@
+<script setup lang="ts">
+import { Category } from '@enums/category.enum'
+</script>
+
 <template>
-  <div>
-    <img :src="imageSrc" alt="Description of the image" class="static-image" />
-  </div>
+  <li>
+    <img :src="getPreviewImageUrl(category, imageSrc)" :alt="imageDesc" class="static-image" />
+  </li>
 </template>
 
-<script>
+<script lang="ts">
 export default {
   name: 'PortfolioPreview',
   props: {
+    category: {
+      type: String,
+      required: true,
+      default: Category.Games,
+      validator: (value: string) => {
+        return Object.values(Category).includes(value as Category)
+      }
+    },
     imageSrc: {
       type: String,
       required: true
+    },
+    imageDesc: {
+      type: String,
+      required: false
+    }
+  },
+  methods: {
+    getPreviewImageUrl(categoryStr: String, imagePath: string): string {
```

Comment:
I think we should make this its own helper function/export and break it out of this component for use elsewhere!

Reply:
Good call!